### PR TITLE
[svg] Replaced Accordion inline svg with css (#7230)

### LINKED ISSUE #7230: [svg] Replace Accordion inline svg with css
inline SVGs are easy, but slow and would benefit from being converted to css

```diff
diff --git a/src/components/shared/Accordion.js b/src/components/shared/Accordion.js
index cd45f6175..534ff03f3 100644
--- a/src/components/shared/Accordion.js
+++ b/src/components/shared/Accordion.js
@@ -69,7 +69,7 @@ class Accordion extends Component<Props, State> {
           onKeyDown={e => this.onHandleHeaderKeyDown(e, i)}
           onClick={() => this.handleHeaderClick(i)}
         >
-          <Svg name="arrow" className={opened ? "expanded" : ""} />
+          <button className={classnames("arrow", { expanded: opened })} />
           {item.header}
           {item.buttons ? (
             <div className="header-buttons" tabIndex="-1">
diff --git a/src/components/shared/Svg.css b/src/components/shared/Svg.css
index 15d2801db..aa43d7b5f 100644
--- a/src/components/shared/Svg.css
+++ b/src/components/shared/Svg.css
@@ -200,3 +200,20 @@ html[dir="rtl"] .arrow svg,
 .source-icon svg {
   fill: var(--theme-comment);
 }
+
+img.arrow, button.arrow {
+  mask: url(/images/arrow.svg) no-repeat;
+  mask-size: 100%;
+  width: 9px;
+  height: 9px;
+}
+
+html[dir="rtl"] .tree-node img.arrow,
+html[dir="rtl"] .tree-node button.arrow {
+  transform: rotate(90deg);
+}
+
+.tree-node img.arrow.expanded,
+.tree-node button.arrow.expanded {
+  transform: rotate(0deg);
+ }
```

---
screen shot 2018-11-05 at 9 59 06 am

### PR DESCRIPTION
**Pull request by acnair**
Fixes #7230 

Here's the Pull Request Doc
https://firefox-dev.tools/debugger.html/docs/pull-requests.html

### Summary of Changes

* Made all the changes as mentioned in issue #7230 

### Test Plan

* Ran ```yarn run jest -u```
* After runing the above the changes were made and were marked as .snap files.

Here's the Debugger's Testing doc
https://docs.google.com/document/d/1oBMRxV8A2ag2t22YsQOxTdEv0mXKzIg0tggJjRkU1S0/edit#.
Feel free to improve it!

### CODE REVIEW

File: src/components/shared/Accordion.js [hunk 1]
```diff
@@ -5,6 +5,7 @@
 // @flow
 import React, { cloneElement, Component } from "react";
 import Svg from "./Svg";
```
Comment (AnshulMalik) [inline src/components/shared/Accordion.js]: we don't need this Svg anymore

**Comment by darkwing** [comment]
A few things:

1.  I'm not seeing the arrow in the button at any state
2. The arrow needs to animate.
3. I used background-image instead of mask and saw the image, so you may want to explore that.  Otherwise, any reason you didn't use an ``?

**Comment by AnshulMalik** [comment]
@acnair Any updates on this?

**Comment by darkwing** [comment]
As discussed with @acnair on Slack, this will get picked up when end of semester exams are done. 👍 

Feel free to open this PR back up when it's ready.  Thank you @acnair !

**Pull request closed without merging**